Formula/freeradius-server.rb

```diff
@@ -29,9 +30,13 @@ class FreeradiusServer < Formula
     sha256 x86_64_linux:   "768b37acaefc41960a324a7bf1f1ca41459c44c90a45e4bbd9a05f6bfdf571ca"
   end
 
+  depends_on "collectd"
   depends_on "openssl@1.1"
   depends_on "talloc"
 
+  uses_from_macos "krb5"
+  uses_from_macos "libpcap"
```

Comment:
These new dependencies seem like a change worth commenting on.

Reply:
I will update the commit message here.